content/product/internal-developer-platforms.md

```diff
@@ -7,7 +7,7 @@ meta_image: /images/product/idp-meta.png
 
 heading: Internal Developer Platform
 subheading: |
-    Build your IDP with Pulumi. Give engineers self-service infrastructure through templates, components, and developer portals while maintaining control through policies and governance.
+    The fastest, most secure way to deliver cloud infrastructure
```

Comment:
Are we sure this is what it said prior to these changes? I vaguely recall we wanted the more descriptive version.